crates/sui-analytics-indexer/src/store/bq/schemas/wrapped_object.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS chaindata.WRAPPED_OBJECT
```

Comment:
IIRC this table will contain more than just objects (i.e. structs with ids) but also wrapped structs in general. If so, would something like wrapped_structs be a better and more clear name? As mentioned in our offline discussions.

Reply:
Thank you for keeping me honest (sorry I totally forgot) and happy to make this change!